src/stk/molecular/topology_graphs/cof/vertices.py

```diff
@@ -310,14 +310,6 @@ class UnaligningVertex(_CofVertex):
 
     """
 
-    def __init__(self, vertex):
```

Comment:
Why is this being removed?

Reply:
It was a previous request for handling unaligning vertices properly to fix all the init methods to not take the vertex, but instead to be a normal vertex init method.